### ISSUE TITLE
[Suggestions] Development progress suggestions

### ISSUE THREAD

**Issue by Jenny77** [DaFuqs/Spectrum#153]
After nearly a year of development tracking, I found that there's a lot of development on some unnecessary stuff,such as those gourmet and tools. I hope there would be more development on the main progression line, and I hope that Deeper Down could be finished... at least, a little.
And I'm happy to help in the development of Deeper Down, for any textures, items, blocks, even sound and music.

**Comment by DaFuqs** [comment]
I would be immensely happy about any help whatsoever!
 Getting the Deeper Down up and running with biomes, blocks and populated by entities is a gigantic undertaking that will take forever solo (Azzy wanted to do the biomes, but is busy with another project right now).

We mostly coordinate stuff to do via Discord and Clickup, so if you want to, swing by the Spectrum Discord at https://discord.com/invite/EXU9XFXT8a, text me and I will invite you to the dev are where you will have a wide range of stuff to choose from, what interests you most. 👍

**Comment by Jenny77** [comment]
Though I've got some problems about signing into Discord, I'm still hyperactive to help!
Sadly my programming abilities are far not enough to do that, but all I can do is to provide some ideas, textures, sound and music! I'd be really happy if I helped someone, and the feeling of being needed is what makes me feel warm and happy.
(PS: I was fishing in the lava in the forever warm Nether, happily with a Molten Rod and relax with my favourite Jade Vine Jelly tea, when I got your replies)
